root/components/EntityTabs.js

```diff
@@ -17,19 +17,18 @@ import Tabs from './Tabs';
 import EntityTabLink from './EntityTabLink';
 
 const tabLinkNames = {
-  'artists': N_l('Artists'),
-  'cover-art': N_l('Cover Art'),
-  'discids': N_l('Disc IDs'),
-  'events': N_l('Events'),
-  'fingerprints': N_l('Fingerprints'),
-  'labels': N_l('Labels'),
-  'map': N_l('Map'),
-  'performances': N_l('Performances'),
-  'places': N_l('Places'),
-  'recordings': N_l('Recordings'),
-  'releases': N_l('Releases'),
-  'users': N_l('Users'),
-  'works': N_l('Works'),
+  artists: N_l('Artists'),
```

Comment:
Does eslint actually complain if none of the keys need quotes? (Otherwise I'd probably keep them, 'cause it doesn't seem unlikely that we'll have others with hyphens.)

Reply:
Yup. "Properties shouldn't be quoted as all quotes are redundant." (quote-props)